package-res/resources/messages/messages_ja.properties

```diff
@@ -1,2 +1,2 @@
-# messagebundleid is the id of this message bundle - do not localize this
-messagebundleid=commons-ui
+# messagebundleid is the id of this message bundle - do not localize this
+messagebundleid=\u5171\u901a UI
```

Comment:
The messagebundleid is NOT supposed to be localized as per the comment on the preceding line.